Performance/SimplePerf/SimplePerf.ps1

```diff
@@ -0,0 +1,325 @@
+            Write-Host "Filters resulted in 0 counters to collect."
+            return
+        }
+
+        $counterFullNames = $countersFiltered | ForEach-Object { ("\\localhost" + $_) }
+
+        if ($DisplayFilterResults) {
+            Write-Host "$($env:COMPUTERNAME): The following counters matched the specified filters:"
+            $counterFullNames | ForEach-Object { Write-Host "$($env:COMPUTERNAME): $_" }
+        }
+
+        $counterFile = (Join-Path $env:TEMP "SimplePerf$($CollectorName)-counters.txt")
+
+        $counterFullNames | Out-File $counterFile
+
+        $OutputFolder = Join-Path $OutputFolder ("$($CollectorName)$([DateTime]::Now.ToString("yyMMddhhmmss"))")
+
+        Write-Host "$($env:COMPUTERNAME): Creating SimplePerf$($CollectorName) collector, writing to $OutputFolder."
+
+        [IO.Directory]::CreateDirectory($OutputFolder) | Out-Null
+
+        $OutputFile = "SimplePerf$($CollectorName)-" + $env:COMPUTERNAME + ".blg"
+
+        if ($Circular) {
+            logman create counter -n "SimplePerf$($CollectorName)" -cf $counterFile -rf $Duration -si $Interval -max $MaximumSizeInMB -o (Join-Path $OutputFolder $OutputFile) -f bincirc
+        } else {
+            logman create counter -n "SimplePerf$($CollectorName)" -cf $counterFile -rf $Duration -si $Interval -max $MaximumSizeInMB -o (Join-Path $OutputFolder $OutputFile) -f bin -cnf 0
+        }
+
+        Write-Host "$($env:COMPUTERNAME): Starting SimplePerf$($CollectorName) collector."
+
+        logman start "SimplePerf$($CollectorName)"
+    }
+
+    function StopSimplePerf {
+        Write-Host "$($env:COMPUTERNAME): Stopping SimplePerf$($CollectorName)."
+        logman stop "SimplePerf$($CollectorName)"
+    }
+
+    $computerTargets = New-Object System.Collections.ArrayList
+}
+
+process {
+    foreach ($computer in $ComputerName) {
+        [void]$computerTargets.Add($computer)
+    }
+}
+
+end {
+    if (-not $Start -and -not $Stop) {
+        Write-Host "Either -Start or -Stop must be specified."
+        return
+    }
+
+    if ($Scenario -eq "None" -and $IncludeCounters.Count -eq 0) {
+        Write-Host "-IncludeCounters is required when Scenario is None."
+        return
+    }
+
+    $argumentList = @(
+        $Duration,
+        $Interval,
+        $MaximumSizeInMB,
+        $OutputFolder,
+        @(GetScenarioDefaults -Scenario $Scenario -Include),
+        @(GetScenarioDefaults -Scenario $Scenario -Exclude),
+        $IncludeCounters,
+        $ExcludeCounters,
+        $Circular,
+        $CollectorName,
+        $($VerbosePreference -ne 'SilentlyContinue')
+    )
+
+    if ($computerTargets.Length -gt 0) {
+        foreach ($computer in $computerTargets) {
+            if ($Start) {
+                Invoke-Command -ComputerName $computer -ScriptBlock ${function:StartSimplePerf} -ArgumentList $argumentList
+            } elseif ($Stop) {
+                Invoke-Command -ComputerName $computer -ScriptBlock ${function:StopSimplePerf}
```

Comment:
This isn't going to work if a different `$CollectorName` is used.

Reply:
Fixed.

![image](https://user-images.githubusercontent.com/4518572/155157106-7cabe4bd-62af-4d93-bd53-9257d4020311.png)